### ISSUE TITLE
Question on .reqif Capella importation (Model Update Window)

### ISSUE THREAD

**Issue by anto761** [eclipse-capella/capella-requirements-vp#189]
Hi,

I am currently trying to import a .reqif file into Capella. My Capella setup is ready for the import: add-ons are installed, Capella Requirements is referenced in the VP Manager, and the Project Explorer filter is configured to display my requirements.

I am using the [pyreqif](https://github.com/ebroecker/pyreqif) Python code to convert my .xlsx file into a .reqif file. The conversion seems to work, as I obtain an XML file that resembles a typical .reqif file.

However, when I try to import the .reqif file into one of my architecture models, the Model Update Window opens, but nothing happens on the right side (see screenshot below—my .reqif file is the one highlighted in bold dark red).

<img width="1401" alt="image" src="https://github.com/eclipse-capella/capella-requirements-vp/assets/173175554/e1a1970f-1795-4ea4-943c-b6d712829b3d">

I suspect there might be a formatting issue within my .reqif file that is preventing a successful import.

As I am relatively new to ReqIF concepts, I am stuck and would greatly appreciate any help or advice.

Thank you,
AP

**Comment by CastroHe** [comment]
Hi,

Have you activated the requirements view on the semantic browser?

A terça, 18/06/2024, 21:14, anto761 ***@***.***> escreveu:

**Comment by anto761** [comment]
Hi,
If you're talking about that, yes I have. 
<img width="160" alt="image" src="https://github.com/eclipse-capella/capella-requirements-vp/assets/173175554/4cd068e5-0ead-44ac-ac34-b6bfad5296fe">

**Comment by CastroHe** [comment]
No, please check the "Requirements in semantic browser and project
explorer" image and text above in paragraph: Requirements attributes viewing
<https://iexcelarc.com/how-to-capture-textual-requirements-in-capella-complete-step-by-step-guide/#:~:text=Requirements%20attributes%20viewing>

Do let me know if that works for you.

**Comment by anto761** [comment]
Hi,

I've attempted to check it, but nothing is appearing in the Semantic Browser window (cf. screenshot).

Since yesterday, I managed to import my .reqif file successfully, and I can see a list of all my requirements. However, the list is empty (cf. screenshot). So, I suspect there might be an issue with the XML file formatting via the conversion file mentioned in my first message, or possibly a problem with the formatting of my original Excel file. But how should I proceed?

<img width="813" alt="image" src="https://github.com/eclipse-capella/capella-requirements-vp/assets/173175554/1ecb508f-a1c6-46c1-9809-89a7c2a51f4b">

Thanks,
AP
